packages/frontend-api/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="frontend_api">
+    <property name="path.openssl.executable" value="openssl"/>
+    <property name="path.defuse-key-generate.executable" value="${path.bin}/generate-defuse-key"/>
+
+    <target name="frontend-api-generate-new-keys" description="Enable frontend API for all domains.">
+        <exec executable="${path.openssl.executable}" checkreturn="true">
+            <arg value="genrsa"/>
+            <arg value="-out"/>
+            <arg value="${path.config}/frontend-api/private.key"/>
+            <arg value="2048"/>
+        </exec>
+        <exec executable="${path.openssl.executable}" checkreturn="true">
+            <arg value="rsa"/>
+            <arg value="-in"/>
+            <arg value="${path.config}/frontend-api/private.key"/>
+            <arg value="-pubout"/>
+            <arg value="-out"/>
+            <arg value="${path.config}/frontend-api/public.key"/>
+        </exec>
+        <exec executable="${path.defuse-key-generate.executable}" checkreturn="true" outputProperty="encryption_key" error="/dev/null">
+        </exec>
+
+        <echo file="${path.config}/frontend-api/.encryption_key" append="false">${encryption_key}</echo>
+    </target>
+
+    <target name="frontend-api-enable" depends="frontend-api-generate-new-keys" description="Enable frontend API for all domains.">
+        <copy file="${path.config}/packages/frontend_api.yaml.dist" tofile="${path.config}/packages/frontend_api.yaml" overwrite="true">
+        </copy>
+    </target>
+
+    <target name="frontend-api-disable" description="Disable frontend API for all domains.">
+        <delete file="${path.config}/frontend-api/.encryption_key" />
```

Comment:
Can I ask why is this file necessary (`.encryption_key`)?

Reply:
Really I don't know. I suppose for some extra key operations, such as recovering a lost public key?

Reply:
From what I found, `generate-defuse-key` is part of https://github.com/defuse/php-encryption package and it's script which generates a random encryption key. 
I don't think it's somehow related to generated keys with openssl.
I tried to remove the file and tokens were generated and used properly. 

If we don't have a use-case for this file, I suggest to remove it as it may confuse people.

Reply:
ok, I removed it 2a65f09bd6b7df3913837a665f091bc3a4f32cfa